bin/rspecq

```diff
@@ -203,5 +203,5 @@ else
   worker.reproduction = opts[:reproduction]
   worker.tags = opts[:tags]
   worker.junit_output = opts[:junit_output]
-  worker.work
+  exit worker.work
```

Comment:
happen when opts[:report] is true?

Reply:
when opts[:report]=true already exits with 1 if there are any leftover failure, [code](https://github.com/moneyforward/rspecq/blob/master/lib/rspecq/reporter.rb#L66). It uses [this build_successful?](https://github.com/moneyforward/rspecq/blob/master/lib/rspecq/queue.rb#L281-L283) condition to check, but if we use this condition on worker mode, then every worker will exit 1 (...I think)